src/management/index.js

```diff
@@ -2015,27 +2015,33 @@ utils.wrapPropertyMethod(ManagementClient, 'getJob', 'jobs.get');
 
 /**
  * Given a path to a file and a connection id, create a new job that imports the
- * users contained in the file and associate them with the given connection.
+ * users contained in the file or JSON string and associate them with the given
+ * connection.
  *
  * @method    importUsers
  * @memberOf  module:management.ManagementClient.prototype
  *
  * @example
  * var params = {
  *   connection_id: '{CONNECTION_ID}',
- *   users: '{PATH_TO_USERS_FILE}'
+ *   users: '{PATH_TO_USERS_FILE}',
+ *   upsert: true, //optional
```

Comment:
I'd remove these optional properties from the sample snippet and keep them in the params docs below

Reply:
Fixed by https://github.com/auth0/node-auth0/pull/469/commits/405cc51ee20534b694837c9fd2f1ec462f064de2. I also wrote about `users_json` as an alternative in https://github.com/auth0/node-auth0/pull/469/commits/0cb767d63ba95051dd9614d412ef0661d731436e.